src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -39,16 +39,18 @@ public void Keyboard_Textbox_InsideScrollViewer_Validation()
 			Assert.AreEqual(string.Empty, multilineTextBox.GetDependencyPropertyValue("Text")?.ToString());
 			Assert.AreEqual(string.Empty, numberTextBox.GetDependencyPropertyValue("Text")?.ToString());
 
+			var initialScreenshot = TakeScreenshot("Initial state");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'initialScreenshot'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4978073)

Reply:
@manpreetkaurnventive Yes this variable can be removed since it's not used inside the test